src/solver/modeler/api/include/antares/solver/modeler/api/linearProblemData.h

```diff
@@ -28,20 +28,33 @@
 namespace Antares::Solver::Modeler::Api
 {
 
-class LinearProblemData
+class ILinearProblemData
+{
+    virtual double getData(std::string idTimeSeriesSet,
+                           std::string scenarioGroup,
+                           unsigned int scenario,
+                           unsigned int hour) = 0;
+};
+
+class LinearProblemData : public ILinearProblemData
```

Comment:
About where we should put class **LinearProblemData** (which is a implementation of interface **ILinearProblemData**) : 
We should take [this page](https://gopro-collaboratif.rte-france.com/pages/viewpage.action?pageId=424772626) (see part 3) into account.
Possible candidate are : 
- **src/study/data-series**
- **src/data-series**

Reply:
With legacy code?
I’d rather create a specific implementation in that place.

Reply:
We have a target of a new source file architecture [here](https://gopro-collaboratif.rte-france.com/pages/viewpage.action?pageId=424772626).